### PR TITLE
Make the overall structure of the screens

### LINKED ISSUE #2: Make the overall structure of the screens
# REQUIREMENTS

Make the navigation structure of screens, the secuence would be: Log Int or Sign Up, my flights and create new flight. 

# Acceptance Criteria
List all required acceptance criteria
- [x] Log In.
- [x] Sign Up.
- [x] My flights.
- [x] Create new flight.

### PR DESCRIPTION
**Pull request by DiegoSpielzeug**
# Description
- We create the files for the Log In, MyFlights and Create new flights screens (and their respective screens) for navigation between the screens. 
- We corrected the paths of the images of the README file of the project, since the folder of "img" was moved to that of "flights":
  
Co-authored-by: JacquelineFM <jfloresm.dev@gmail.com>

### CODE REVIEW

**Review by YilKanda — CHANGES_REQUESTED** [review]
Verify that the CodeClimate gives you approval, apparently it marks 7 files as similar codes. Can 

I recommend using a different structure, for example in one use a View and in another a SafeView, so that it does not mark an error.

imagen

**Comment by Seal-Trainer** [comment]
I don't agree with @YilKanda's opinion, _CodeClimate_ **can be wrong**, recently I was working on creating a database and it _marked as error_ the structure (id, name, etc), because it was from a JSON _because it was repeated_, which makes sense that the structure is repeated, if the only thing they will do is to change the tags so that CodeClimate does not find it as repeated, I would choose to mark in CodeClimate those "errors" as "**invalid errors**" (if you do not know how to do this I can explain you), that is my opinion, I do not know what you think YilKanda.

File: flights/FlightApp.js [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
 
-const Stack = createStackNavigator()
+const Stack = createStackNavigator();
 
 const FlightApp = () => {
   return (
-    <NavigationContainer >
-      <Stack.Navigator>
+    <NavigationContainer>
+      <Stack.Navigator initialRouteName="SingUp">
```
Comment (Seal-Trainer) [inline flights/FlightApp.js]: Shouldn't this be "SignUp"?

**Comment by YilKanda** [comment]
> I don't agree with @YilKanda's opinion, _CodeClimate_ **can be wrong**, recently I was working on creating a database and it _marked as error_ the structure (id, name, etc), because it was from a JSON _because it was repeated_, which makes sense that the structure is repeated, if the only thing they will do is to change the tags so that CodeClimate does not find it as repeated, I would choose to mark in CodeClimate those "errors" as "**invalid errors**" (if you do not know how to do this I can explain you), that is my opinion, I do not know what you think YilKanda.

I agree that CodeClimate shows that invalid error, anyway, it was just a possible suggestion to always keep a good CodeClimate rating constant.

File: flights/FlightApp.js (hunk 1 above)
Comment (JacquelineFM) [reply flights/FlightApp.js]: Hahahahahahaha, right! Finger error. Thanks a lot! We'll modify it.

Comment (JacquelineFM) [reply flights/FlightApp.js]: Ready! We've already modified it! @Seal-Trainer

**Review by YilKanda — APPROVED** [review]

**Review by Seal-Trainer — APPROVED** [review]

**Pull request merged**